assets/javascripts/discourse/connectors/advanced-search-options-below/category-experts-search-fields.hbs

```diff
@@ -0,0 +1,27 @@
+<div class="control-group">
+  <label class="control-label">{{i18n "category_experts.title"}}</label>
+  <div class="controls">
+    <section class='field'>
+      <label>
+        {{input
+          type="checkbox"
+          class="with-category-expert-response"
+          checked=(readonly searchedTerms.withCategoryExpertResponse)
+          click=(action "onChangeCheckBox" "withCategoryExpertResponse" "_updateWithCategoryExpertResponse")
```

Comment:
Typically underscores mean private APIs. I think it's unusual to call something in a template that has a private API.

Reply:
yeah, not sure why that's underscored. will undo